src/main/java/org/fungover/mmotodo/team/TeamService.java

```diff
@@ -0,0 +1,124 @@
+package org.fungover.mmotodo.team;
+
```

Comment:
There's some weird formatting going on in this file, I suggest changing your IDE settings to automatically change such things on saving a file.